최재훈/index.js

```diff
@@ -0,0 +1,27 @@
+// 결과 값
+const resultDisplay = document.querySelector(".js-result");
+
+// 내가 입력하는 값
+const inputDisplay = document.querySelector(".js-input");
+
+// AC, Del, = 제외 버튼 클릭 시 inputDisplay에 보여주기 위함
+function add(char) {
+  inputDisplay.value += char;
+}
+
+// "=" 클릭 시 계산 하는 기능
+function equal() {
+  const result = eval(inputDisplay.value);
+  resultDisplay.value = result;
+}
+
+// AC버튼 클릭 시 reset() 함수 실행
+function reset() {
+  document.querySelector(".js-result").value = "";
+  document.querySelector(".js-input").value = "";
```

Comment:
여기서는 위에 선언하신 DOM 요소를 사용하지 않은 이유가 무엇인가요??

```suggestion
  resultDisplay.value = "";
  inputDisplay.value = "";
```

Reply:
불필요한 부분이 있어서 코드 수정해두었습니다 . .